mobile/src/main/res/layout/activity_main.xml

```diff
@@ -36,18 +36,28 @@
             android:background="?attr/colorPrimary"
             android:theme="@style/Base.ThemeOverlay.AppCompat.Dark.ActionBar">
 
-            <FrameLayout
+            <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="match_parent">
+                android:layout_height="match_parent"
+                android:orientation="horizontal">
 
                 <ImageView
                     android:id="@+id/drawer_menu_icon"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
                     android:layout_gravity="start|center_vertical"
-                    android:src="@drawable/ic_toolbar_menu" />
+                    auto:srcCompat="@drawable/ic_toolbar_menu" />
```

Comment:
replaced `android:src` with `auto:srcCompat`.
Otherwise app would crash for earlier versions of android because of vector drawables.